win32ss/user/ntuser/ntstubs.c

```diff
@@ -346,16 +346,57 @@ NtUserSetSysColors(
    return Ret;
 }
 
-DWORD
+BOOL APIENTRY UserUpdateInputContext(PIMC pIMC, DWORD dwType, DWORD_PTR dwValue)
+{
+    PTHREADINFO pti = GetW32ThreadInfo();
+    PTHREADINFO ptiIMC = UserHMGetHandle(pIMC);
```

Comment:
This is strange. PIMC is not a handle, and PTHREADINFO is not a handle either.

Reply:
Corrected to `pIMC->head.pti`.